lib/diagnostic.rb

```diff
@@ -95,7 +117,9 @@ def eat_meat(food)
 # Write your answer as a comment in the section below.
 
 ##
-# your answer, in comments, here
+# Mixins are a great way to use inheritance because you can use specific methods on a class instead of
+# having to call an entire class's set of methods and traits. It also has the advantage of being
```

Comment:
Sort of. It does give you the option of making your methods more modular.
But the biggest advantage to using mixins is that allows for multiple inheritance, since you can load _multiple_ mixins, but can only inherit from a single class (though, admittedly, you also get access to everything that the parent class inherits from).